uk.co.compucorp.civicrm.hrcore/backstop_data/page-objects/ssp-leave-absences.js

```diff
@@ -0,0 +1,32 @@
+var Promise = require('es6-promise').Promise;
+var page = require('./page');
+
+module.exports = (function () {
+
+  return page.extend({
+    /**
+     * [init page]
+     */
+    initPage: function () {
+      var casper = this.casper;
+
+      return new Promise(function (resolve) {
+        casper.then(function () {
+          casper.wait(10000);
```

Comment:
why are you arbitrarily waiting for 10000 seconds (which is a lot)? isn't there something that you can specifically target with a `waitUntilVisibile`?